Example/FirestoreSample/.firebaserc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I believe `Firestore/Swift/Tests/FirestoreSample` would be a more consistent location with the rest of the repo than `Example/FirestoreSample`.